packages/components/src/select-control/control.js

```diff
@@ -122,8 +122,11 @@ class Control extends Component {
 			onSearch,
 			placeholder,
 			searchInputType,
+			multiple,
+			selected,
 		} = this.props;
 		const { isActive } = this.state;
+		const disabled = ! multiple && inlineTags && selected.length > 0;
```

Comment:
You could include `isActive` in this calculation and simplify the logic further on

Reply:
I've removed this code.. I found it to be problematic (keeping `isActive` state but disabling the nested `input`, etc etc) - the UX without disabling the nested `input` isn't all that confusing anyhow.